test-framework/org.jboss.tools.cdk.reddeer/META-INF/MANIFEST.MF

```diff
@@ -8,7 +8,7 @@ Bundle-ActivationPolicy: lazy
 Bundle-Vendor: Red Hat
 Bundle-RequiredExecutionEnvironment: JavaSE-1.8
 Require-Bundle: org.junit,
- org.eclipse.reddeer.go;bundle-version="[2.1.0,3.0.0)",
```

Comment:
see comment above about the version range for reddeer

Reply:
The reason why I removed the upper bounds is that RedDeer switch to 3.0 when we changed the TP thus lots of builds failed but there was no API breakage AT ALL so I prefer discover them when they really EXIST

Reply:
That's right. There is no API breakage but removed unused feature in IU. So it is new version but without a real breakage. I would maybe prefer to leave lower bound (2.1.0) in place, because between version 1 a 2 there is quite a lot breakages... But we control what reddeer version is used (via TP), so... it is at least nice to have...